librpc/Rpc.cpp

```diff
@@ -1007,10 +1007,12 @@ std::string Rpc::sendRawTransaction(int _groupID, const std::string& _rlp)
         if (currentTransactionCallback)
         {
             auto transactionCallback = *currentTransactionCallback;
-            tx.setRpcCallback(
-                [transactionCallback](LocalisedTransactionReceipt::Ptr receipt, bytes input) {
-                    Json::Value response;
-
+            auto clientProtocolversion = (*m_transactionCallbackVersion)();
+            tx.setRpcCallback([transactionCallback, clientProtocolversion](
+                                  LocalisedTransactionReceipt::Ptr receipt, bytes input) {
+                Json::Value response;
+                if (clientProtocolversion > 0)
```

Comment:
what's the meaning of 0 here?

Reply:
Please read the comment. The if is useless for now